src/xcode/ENA/ENA/Source/Services/RecycleBin/RecycleBin.swift

```diff
@@ -56,10 +57,13 @@ class RecycleBin {
 		case .certificate:
 			Log.info("Ask for certificate item restoration.", log: .recycleBin)
 			return .success(())
-		case .coronaTest(let coronaTest):
-			Log.info("Ask for test item restoration.", log: .recycleBin)
-			let canRestoreResult = testRestorationHandler.canRestore(coronaTest)
+		case .userCoronaTest(let coronaTest):
+			Log.info("Ask for user test item restoration.", log: .recycleBin)
+			let canRestoreResult = userTestRestorationHandler.canRestore(coronaTest)
 			return canRestoreResult.mapError { RestorationError.testError($0) }
+		case .familyMemberCoronaTest:
+			Log.info("Ask for family member test item restoration.", log: .recycleBin)
+			return .success(())
```

Comment:
the familiy test can always be restored?
Or is the request for canRestore missing?

Reply:
Yes, they can always be restored, as there is no limitation on the amount of tests as opposed to the user tests